src/pages/index.js

```diff
@@ -5,6 +5,8 @@ import NotificationPage from './NotificationPage/NotificationPage';
 import BookmarksPage from './BookMarksPage/BookmarksPage';
```

Comment:
You can create Index File for import and export.

Reply:
sure, thanks for suggestion.